Base.rte/Devices/Tools/Constructor/Constructor.ini

```diff
@@ -520,47 +554,39 @@ AddAmmo = Magazine
 	JointStiffness = 1
 	JointOffset = Vector
 		X = 0
-		Y = 0
+		Y = -7
+	ParentOffset = Vector
+		X = -1
+		Y = 1
 	DrawAfterParent = 0
 	RoundCount = -1
 	RTTRatio = 0
 	RegularRound = Round
-		PresetName = Round Constructor
-		ParticleCount = 1
-		Particle = MOPixel
-			CopyOf = Null Bullet
-		Shell = None
-		FireVelocity = 100
-		ShellVelocity = 10
-		Separation = 10
-
-
-///////////////////////////////////////////////////////////////////////
```

Comment:
Why remove the separator? 
Helps see when the actual device code begins.

Applies to all other separator deletions.

Reply:
Probably an accident there.